bokehjs/src/coffee/models/glyphs/quad.coffee

```diff
@@ -41,6 +67,23 @@ class QuadView extends Glyph.View
     result['1d'].indices = hits
     return result
 
+  get_anchor_point: (anchor, i, spt) ->
+    left = Math.min(@sleft[i], @sright[i])
+    right = Math.max(@sright[i], @sleft[i])
+    top = Math.min(@stop[i], @sbottom[i])     # screen coordinates !!!
```

Comment:
That's the idea with view vs screen coords but it seems there is some leakage. 

> On May 26, 2016, at 06:18, Mateusz Paprocki <notifications@github.com> wrote:
> 
> In bokehjs/src/coffee/models/glyphs/quad.coffee:
> 
> > @@ -41,6 +67,23 @@ class QuadView extends Glyph.View
> >      result['1d'].indices = hits
> >      return result
> >  
> > +  get_anchor_point: (anchor, i, spt) ->
> > +    left = Math.min(@sleft[i], @sright[i])
> > +    right = Math.max(@sright[i], @sleft[i])
> > +    top = Math.min(@stop[i], @sbottom[i])     # screen coordinates !!!
> We need proper abstractions to deal with coordinate systems (among other things). I shouldn't be concerned with whether top is the minimum or maximum of a set of values in screen coordinates.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>